### PR TITLE
Add Redis Cache Deployment To Deploy-All

### PR DESCRIPTION
**Pull request by dagrooms52**

### CODE REVIEW

File: infrastructure/ArmTemplates/DeployAllTemplate.json [hunk 1]
```diff
@@ -103,105 +96,128 @@
                     "siteName": "[replace(parameters('RuntimeHost1'),'1','2')]"
                 }
             ]
+        },
+        "RedisCache": {
+            "Name": "LiberationCache-Prod",
```
Comment (SharmaAjay19) [inline infrastructure/ArmTemplates/DeployAllTemplate.json]: Can we name it something like diagnostics cache or app service diagnostics cache?

Comment (dagrooms52) [reply infrastructure/ArmTemplates/DeployAllTemplate.json]: That sounds better, I'll do DiagnosticsCache

File: infrastructure/ArmTemplates/DeployAllTemplate.json [hunk 2]
```diff
@@ -103,105 +96,128 @@
                     "siteName": "[replace(parameters('RuntimeHost1'),'1','2')]"
                 }
             ]
+        },
+        "RedisCache": {
+            "Name": "DiagnosticsCache-Prod",
+            "diagnosticsEnabled": true,
+            "existingDiagnosticsStorageAccountId": "/subscriptions/0e7b3183-4428-4c1f-93cd-cb2a26616a8a/resourceGroups/RedisCache-RG/providers/Microsoft.Storage/storageAccounts/storeokauaj64yhhns"
```
Comment (ShekharGupta1988) [inline infrastructure/ArmTemplates/DeployAllTemplate.json]: Possible to not checkin the exact resource uri here?

Comment (dagrooms52) [reply infrastructure/ArmTemplates/DeployAllTemplate.json]: I'm not sure about that. I'm not even sure this template runs. We could probably delete this PR.

**Pull request closed without merging**